File: src/main/java/com/nickzim/DTOs/Message.java
package com.nickzim.DTOs;

import com.nickzim.DTOs.Attachments.Attachment;
import lombok.Data;

@Data
public class Message {

    private Integer id;

    private Integer date;

    private Integer peer_id;

    private Integer from_id;

    private String text;

    private Integer random_id;

    private String ref;

    private String ref_source;

    private Attachment[] attachments;

    private Boolean important;

    private Message[] fwd_messages;

    private Integer conversation_message_id;

}

File: src/main/kotlin/com/nickzim/domain/Message.kt
package com.nickzim.domain

import com.nickzim.domain.attachments.Attachment

data class Message(
        val id: Int?,
        val date: Int?,
        val peer_id: Int?,
        val from_id: Int?,
        val text: String?,
        val random_id: Int?,
        val ref: String?,
        val ref_source: String?,
        val attachments: Array<Attachment>?,
        val important: Boolean?,
        val fwd_messages: Array<Message>?,
        val conversation_message_id: Int?

)
